README.md

```diff
@@ -74,7 +74,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.willowtreeapps:signinwithapplebutton:0.4-SNAPSHOT'
+    implementation 'com.willowtreeapps:signinwithapplebutton:0.3-SNAPSHOT'
```

Comment:
Normally the SNAPSHOT would be for the next version, might want to set the current repo version to `0.4-SNAPSHOT` and list that here.